sla/sla-core/sla-personalization/pom.xml

```diff
@@ -53,6 +59,11 @@
 			<groupId>com.sun.jersey</groupId>
 			<artifactId>jersey-client</artifactId>
 		</dependency>
+
+		<dependency>
+			<groupId>it.polimi.modaclouds.qos-models</groupId>
+			<artifactId>qos-models</artifactId>
+		</dependency>
```

Comment:
Here is the one I'm currently using in the monitoring-dam-generator 

        <dependency>
        <groupId>it.polimi.tower4clouds</groupId>
        <artifactId>rules</artifactId>
        <version>0.2.3</version>
    </dependency>    

Of course you need also the repository declaration.